app/controllers/submit.js

```diff
@@ -247,8 +252,10 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
             this.set('node.title', this.get('nodeTitle'));
             let node = this.get('node');
             node.save();
+            Ember.run.scheduleOnce('afterRender', this, function() {
+                MathJax.Hub.Queue(['Typeset', MathJax.Hub, Ember.$('.preprint-header-preview')[0]]);  // jshint ignore:line
```

Comment:
Not for this PR- but eventually, we'll want to DRY these selector names in one place when we clean up this page. 

This code is now closely tied to the markup of a template (just changing a style tag could affect how/if mathjax renders!), so we'll want to task this as part of that future cleanup.